guides/plugins/plugins/administration/add-mixins.md

```diff
@@ -0,0 +1,69 @@
+# Adding Mixins
+
+This documentation chapter will cover how to add a new administration Mixin for your plugin.
+Mixins behave generally the same as they do normally in Vue, just the registration and how mixins are included in a component is different.
+
+## Prerequisites
+
+All you need for this guide is a running Shopware 6 instance and full access to both the files and a running plugin. Of course you'll have to understand JavaScript, but that's a prerequisite for Shopware as a whole and will not be taught as part of this documentation. As stated before mixins in Shopware are basically the same as in Vue so you should have read their [documentation](https://vuejs.org/v2/guide/mixins.html) on them first. 
+
+## Register a new service
+
+For this example, we'll just use the example mixin from the [VueJS documentation](https://vuejs.org/v2/guide/mixins.html) and adjust it to be used in Shopware.
+
+Mixins in Shopware have to be registered in the mixin registry via the `Mixin.register` function to available everywhere in the administration.
+
+Converting the Vue mixin to be used in Shopware and looks like the example seen below:
+
+{% code title="<administration root>/mixins/swag-basic-example.js" %}
+```javascript
+const { Mixin } = Shopware;
+
+Mixin.register('swag-basic-mixin', {
+    created: function () {
+        this.hello()
+    },
+    methods: {
+        hello: function () {
+            console.log('hello from mixin!')
+        }
+    }
+});
+```
+{% endcode %}s
+
+## Importing the Mixin it the Plugin
+
+Now that we have registered the mixin, we have to import it at the top of our `main.js`.
+
+```javascript
+{% code title="<administration root>/src/main.js"%}
+import '<administration root>/mixins/swag-basic-example.js'
+{% endcode %}
+```
+
+## Using the Mixin
+
+Now that we have registered our mixin under a name, we can get it form the registry with the `Mixin.getByName` function and inject into our component as seen below.
+
+{% code title="<administration root>/components/swag-basic-example/index.js" %}
+```javascript
+const { Component, Mixin } = Shopware;
+
+Component.register('swag-basic-example', {
+
+    mixins: [
+        Mixin.getByName('swag-basic-mixin')
+    ],
+});
+```
+{% endcode %}
+
+## Next steps
+
+Now that we have created a mixin, you might want to create or customize a administration component:
+* [Creating a new administration component](./add-custom-component.md)
+* [Extending an existing administration component](./customizing-components.md)
+
+Or you might want to add your own service to the administration:
```

Comment:
```suggestion
Another possibility is adding your own service to the administration:
```

Reply:
It would like to leave this as is, because adding services dosn't really have anything to do with the other 2 points above

Reply:
I don't like the `Or` in the beginning. Let's search for an alternative then, e.g 
```It may also be that you want to add...```

Reply:
Then maybe a compromise would be:

```suggestion
If you however want to add your own service to the administration:
```

Reply:
I think that is great